src/app/token-option-field-component-factories/spend-for-quiz-revision-token-option-field-component-factory.ts

```diff
@@ -66,7 +66,7 @@ export class SpendForQuizRevisionTokenOptionFieldComponentFactory extends TokenO
                 .appendBuilder(
                     createNewDueTimeComponentBuilder(
                         environmentInjector,
-                        'Until Time for Explanation Document Submission'
+                        'Change Canvas Assignment’s Available Until to'
```

Comment:
Please see comment on line 62 of `spend-for-assignment-resubmission-token-option-field-component-factory.ts`.